Make comments match demonstrated code (#2072)
microsoft/TypeScript-Website: packages/documentation/copy/en/handbook-v2/Type Manipulation/Template Literal Types.md

person.on("firstNameChanged", () => {});

> // Prevent human error (using a key instead of a signal)

Reviewer: Maybe this? I don't think we use signal elsewhere in this doc

```suggestion
// Prevent easy human error (using the key instead of the event name)
```
